framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -58,7 +65,10 @@ public static void main(String[] args) {
         System.exit(exitStatus);
     }
 
-    /** The path to the annotated jdk jar to use. */
+    /**
+     * In the case of Java 8, the path to the annotated jdk.jar. In the case of Java 9+, the path to
+     * a directory containing annotated-jdk modules.
+     */
     protected final File jdkJar;
```

Comment:
I suggest that you rename this `jdkPath` because the current name is not accurate for JDK 9+.

Reply:
Done.